src/Essentials/src/Geolocation/Geolocation.ios.macos.cs

```diff
@@ -53,9 +60,22 @@ public async Task<Location> GetLocationAsync(GeolocationRequest request, Cancell
 
 			manager.StartUpdatingLocation();
 
+			var reducedAccuracy = false;
+#if __IOS__
+            if (OperatingSystem.IsIOSVersionAtLeast(14, 0))
+            {
+                if (request.RequestFullAccuracy && manager.AccuracyAuthorization == CLAccuracyAuthorization.ReducedAccuracy)
+                {
+                    await manager.RequestTemporaryFullAccuracyAuthorizationAsync("XamarinEssentialsFullAccuracyUsageDescription");
```

Comment:
Is it possible to use a different name? One without the xamarin.essentials, and maybe one that fits better with OS ones just so it looks less out of place?

Reply:
Good call! Updated it to just `TemporaryFullAccuracyUsageDescription`?